### ISSUE TITLE
Hotkey for deleting checkpoints?

### ISSUE THREAD

**Issue by rjp5nc** [awedwards/daphnia#23]
Could make it so that if we press a key (i.e. q) it would be the same as pressing the  "Delete checkpoint" button? There are times where we have to delete several checkpoints in a row, and having the tail and tail tip checkpoints makes it a bit more tedious to delete long rows of them. If we could have a key to do the same, that would speed things up quite a bit more.

**Comment by rjp5nc** [comment]
Probably not Q, as it turns out that that closes the GUI tool, but some other key that is easy to press and unused for anything else

**Comment by awedwards** [comment]
Yeah that makes sense. I'll look into how hard that would be to add, but it should be straightforward.

**Comment by awedwards** [comment]
Added!

It very easy to add hotkeys. Can you think of other ones that might be useful?

Here are some I was thinking about:
- arrow keys for moving checkpoint
- control + arrow keys for cycling through checkpoints (as in, changing the 'selected' checkpoint)

**Comment by rjp5nc** [comment]
Which key was added to be the hotkey for deleting checkpoints? Maybe adding a similar key to enter "add checkpoint" mode would compliment it nicely. 

 I think both of those could be a good idea. Would moving the checkpoint move it via a set number of pixels, or to the next line closest? Between the two, I think having it move a set number of pixels might be nice (for fine tuning, maybe have the arrow keys move it like a few pixels at a time?) Would we run into problems with having the line recalculate itself every time we moved the checkpoint a pixel, though?

**Comment by awedwards** [comment]
Delete and backspace. I did both because I don't have a delete key on my laptop :)

Yeah I might have to test it and see how annoying it would be to recalculate. Maybe we'll want an 'update' button/hotkey...

**Comment by rjp5nc** [comment]
I think that we would prefer not to have to press update every time we actually move a checkpoint, though, as that would probably become fairly tedious. I think we may want to check out how easy it is for it to recalculate it, but if we actually have to manually update it, I think that would take a bit too much time/effort to get it right.

**Comment by awedwards** [comment]
Yeah, that's totally fair. Ok I'll check out how moving a pixel behaves and also a hotkey for toggling 'add pixel' state.
